README.md

```diff
@@ -175,7 +175,11 @@ pouch info | grep rune
 
 # Deployment
 
-Inclavare Containers can be deployed with dockerd, containerd, and pouchd.
+Inclavare Containers can be deployed with dockerd, containerd, and pouchd. 
+
+[The reference container images](https://hub.docker.com/u/inclavarecontainers) are available for the demonstration purpose. Currently, WebAssembly Micro Runtime (WAMR) application demo and web application demos based on OpenJDK 11, [Dragonwell](http://dragonwell-jdk.io/), and Golang are provided.
+
+**All reference container images don't contain enclave runtime. They are only used for the deployment with dockerd, containerd, and pouchd.**
```

Comment:
Actually the WAMR image contains enclave runtime already, because it doesn't adapt to shim-rune and use off-cloud signing. It is necessary to explicitly list the corresponding relationship between the reference container image and deployment.